scss/6-components/_sidebar.scss

```diff
@@ -0,0 +1,46 @@
+.cads-sidebar {
```

Comment:
Mind adding a `/** @define sidebar */` comment at the top of this file. That will tell the linter to check for BEM conventions.